docs/proposal/3_8_risk_profile.tex

```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
-Software license does not allow our application to be delivered.&0.3&0.6&0.18\\ \hline
-Key hardware components not available&0.2&0.8&0.16\\ \hline
-Machine learning resources lack documentation&0.3&0.5&0.15\\ \hline
-Camera lack documentation&0.3&0.5&0.15\\ \hline
-FPGA lack documentation&0.3&0.5&0.15\\ \hline
-Delivered documentation not adequate for usage and upgrade.&0.5&0.3&0.15\\ \hline
-Selected software methodology creates issues with team member that decreases productivity&0.2&0.7&0.14\\ \hline
-Legal changes significantly affect project --- drones in particular&0.2&0.7&0.14\\ \hline
-The final product cannot be maintained or extended.&0.7&0.2&0.14\\ \hline
-Client is not cooperative or does not provide necessary information&0.2&0.7&0.14\\ \hline
-Client wants to modify scope and requirements that leads to delays or cut features.&0.2&0.6&0.12\\ \hline
-Development environment is inadequate.&0.2&0.6&0.12\\ \hline
-Sabotage&0.1&1&0.1\\ \hline
-Loss of client&0.1&1&0.1\\ \hline
-Loss of team member&0.1&0.9&0.09\\ \hline
-Equipment shipment is delayed or lost.&0.1&0.8&0.08\\ \hline
-Client is not available enough to provide significant help.&0.1&0.5&0.05\\ \hline
-Competitive offerings affect project requirement&0.1&0.5&0.05\\ \hline
-Camera doesn’t work with FPGA&0.1&0.5&0.05\\ \hline
-FPGA doesn’t work with transmission systems&0.1&0.5&0.05\\ \hline
+
+Drone flight hardware (flight controllers, radio, motors) cannot function to crashes and damages. & 0.9 & 1.0 & 0.90 \\ \hline
+Payload is too heavy which significantly increases drone motor requirements and significant reduction in flight duration. & 0.8 & 0.8 & 0.64 \\ \hline
+Accidents that damage the drone and computating equipment that require extra budget that we may not have. & 0.6 & 0.9 & 0.54\\ \hline
+Loss of drone hardware and payload during flight and cannot be recovered (total loss). & 0.5 & 1.0 & 0.50 \\\hline
+Not enough time commitment from team members. & 0.7 & 0.7 & 0.49\\ \hline
+Access to tools and shops for modifiying and repairing drone hardware is inadequate or non-existent. & 0.6 & 0.8 & 0.48\\ \hline
+Underestimation of project scope or work required, leading to insufficient time managemnet and burn-outs. & 0.5 & 0.9 & 0.45\\ \hline
+Payload is too heavy which exceeds total take-off weight. & 0.4 & 1.0 & 0.40 \\ \hline
+Legacy documents for the project is insufficient, resulting in the final product unable to be maintained or extended. & 0.7 & 0.5 & 0.35\\ \hline
+Financial inefficiencies leading to overbudget or lack of capital. & 0.4 & 0.8 & 0.32\\ \hline
+Constrained to purchase lower-quality components due to budget and results in lower performance. & 0.6 & 0.5 & 0.30 \\ \hline
+Team is indecisive or team leader cannot make a timely decision --- results in delay. & 0.4 & 0.6 & 0.30\\ \hline
+Development and managment technique and methodology is not effective, leading to productivity losses. & 0.4 & 0.7 & 0.28\\ \hline
```

Comment:
```suggestion
Development and management technique/methodology is not effective, leading to productivity losses. & 0.4 & 0.7 & 0.28\\ \hline
```

Reply:
What do you have against "and" or "or"s 😠